### PR TITLE
Created the GameCreationInteractor 

### PR DESCRIPTION
**Pull request by susannah-c**
Created the GameCreationInteractor - there are 2 methods both called createGame. One is to create a new game and initialize a new Board while the other is to create a game from saved data for when the users choose to resume a game.

### CODE REVIEW

File: src/main/java/Interactors/GameCreation.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        // This method is to create a brand-new game and initialize a new Board.
+        ArrayList<Player> players = new ArrayList<>();
+        Player player1 = new Player("red");
+        Player player2 = new Player("blue");
+        Player player3 = new Player("green");
+        Player player4 = new Player("yellow");
+        players.add(player1);
+        players.add(player2);
+        players.add(player3);
+        players.add(player4);
```
Comment (giaphuongd) [inline src/main/java/Interactors/GameCreation.java:23]: Is this assuming there will be exactly 4 players in each game? I'm pretty sure it's 2-8 players. This implementation disregards the case where there can be more or less than 4 players. Also I'm assuming the players may want to choose their own names (but this is less important).

File: src/main/java/Entities/Player.java [hunk 2]
```diff
@@ -156,7 +156,8 @@ public void load(File pw) {
         }
 
     }
-    private void createPlayer(ArrayList<ArrayList> All){
```
Comment (giaphuongd) [inline src/main/java/Entities/Player.java:159]: createPlayer was a helper function for load (which was originally in Player). We have moved load to a different class and removed createPlayer so you can also get rid of this

**Review by giaphuongd — CHANGES_REQUESTED** [review]
Try to talk to the other classes for GameCreation. I think someone is handling the input of how many players there will be, then you can implement createGame with varying number of players instead of just 4. Also try not to push changes you made to classes that are being coded by other people e.g. Player. If you think there are changes needed in those classes, you can talk to people who are in charge of them :)

File: src/main/java/Interactors/GameCreation.java (hunk 1 above)
Comment (derrcodes) [reply src/main/java/Interactors/GameCreation.java:23]: Yeah we have to fix this. I think you should talk to the people handling the user input to coordinate how exactly the details on the Player instances will be made.

**Review by derrcodes — CHANGES_REQUESTED** [review]
Some changes need to be made that have already been pointed out by Zach like the number of players currently being limited to 4 and just 4. I see there are also changes to the Board class however the constructor needs to be changed together with the Property class as I believe it has been refactored as shown in the Board branch.

File: src/main/java/Entities/Property.java [hunk 3]
```diff
@@ -93,7 +93,7 @@ public int getHouses(){
         return this.houses;
     }
 
-    public ArrayList<String[]> loadProperties() throws FileNotFoundException {
+    public static ArrayList<String[]> loadProperties() throws FileNotFoundException {
         // return array of all properties in txt file as Strings
         // GameCreationInteractor will parse Strings to create Property instances
         ArrayList<String[]> allProperties = new ArrayList<>();
```
Comment (Cindywwang) [inline src/main/java/Entities/Property.java:99]: Is loadProperties() only returning an empty list? Where does the property info come from?

**Pull request closed without merging**